.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "PartCAD Dev Container",
+  "build": { // 13:27.664
+    "dockerfile": "./Dockerfile"
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/python:1": { // 06:25.200
+      // https://github.com/devcontainers/features/tree/main/src/python
+      "version": "3.10",
+      "installTools": true,
+      "toolsToInstall": "pre-commit,poetry",
+      "optimize": true,
+      "enableShared": false,
+      "installPath": "/usr/local/python",
+      "installJupyterlab": false,
+      "configureJupyterlabAllowOrigin": "",
+      "httpProxy": ""
+    },
```

Comment:
- [ ] Check how to have this added to Image in different workflow so we compile Python only once.